packages/curve-ui-kit/src/shared/ui/Metric.tsx

```diff
@@ -0,0 +1,195 @@
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { SizesAndSpaces } from 'themes/design/1_sizes_spaces'
+import { Fonts } from 'themes/typography/fonts'
```

Comment:
Fonts are already defined in the Typography, they should change depending on the theme and the variant